typescript plugin still references rollup-pluginutils

<!--
  ⚡️ katchow! We 💛 issues.

  Please - do not - remove this template.
  Please - do not - skip or remove parts of this template.
  Or your issue may be closed.

  👉🏽 Need help or tech support? Please don't open an issue!
  Head to https://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs

  ❤️ Rollup? Please consider supporting our collective:
  👉 https://opencollective.com/rollup/donate
-->

- Rollup Plugin Name: typescript
- Rollup Plugin Version: 2.0.0
- Rollup Version: 1.27.8
- Operating System (or Browser): macOS
- Node Version: 12.13.1

### How Do We Reproduce?

<!--
  Issues without minimal reproductions will be closed! Please provide one by:
  1. Using the REPL at https://rollupjs.org/repl/, or
  2. Work to isolate the problem and provide the exact steps in this issue, or
  3. Provide a minimal repository link (Read https://git.io/fNzHA for instructions).
     These may take more time to triage than the other options.
-->

- Install the typescript plugin `npm i --save-dev @rollup/plugin-typescript typescript tslib`
- Add it to _rollup.config.js_
- Run  `rollup -c`

### Expected Behavior

It should work ;) 

### Actual Behavior

```
> rollup -cw

[!] Error: Cannot find module 'rollup-pluginutils'
Require stack:
- /Users/lukas/Documents/Bachmann/atscm/transformer-help/node_modules/@rollup/plugin-typescript/dist/index.js
- /Users/lukas/Documents/Bachmann/atscm/transformer-help/rollup.config.js
- /Users/lukas/Documents/Bachmann/atscm/transformer-help/node_modules/rollup/dist/bin/rollup
Error: Cannot find module 'rollup-pluginutils'
```

<!--
  Most issues can be expressed or demonstrated through the REPL or a repository.
  However, the situation may arise where some small code snippets also need to
  be provided. In that situation, please add your code below using
  Fenced Code Blocks (https://help.github.com/articles/creating-and-highlighting-code-blocks/)
-->

### Reason

The problem is, that [_index.js_ still imports 'rollup-pluginutils' instead of '@rollup/pluginutils'](https://github.com/rollup/plugins/blob/126c4e374b9123ef7dcab57a3b375082e26a3e4e/packages/typescript/src/index.js#L4).

### Workaround 

Install rollup-pluginutils: `npm i --save-dev rollup-pluginutils`

Are you on this one or should I create a PR?

--- a/packages/typescript/src/index.js
+++ b/packages/typescript/src/index.js
@@ -1,7 +1,7 @@
 import * as fs from 'fs';
 
 import * as ts from 'typescript';
-import { createFilter } from 'rollup-pluginutils';
+import { createFilter } from '@rollup/pluginutils';
 import resolveId from 'resolve';
 
 import endsWith from './string';

